### ISSUE TITLE
[google] wrong URL

### ISSUE THREAD

**Issue by dalf** [searx/searx#491]
With the google search engine, when I search [san francisco restaurants](http://searx.me/?q=%21google+san+francisco+restaurants) for example (any location + restaurants), the first links give restaurants and phone numbers but the URLs are wrong.

The function involved : 
https://github.com/asciimoo/searx/blob/master/searx/engines/google.py#L279

The reason : since few months ( ? ), the no javascript version of google doesn't give the direct link to web site, but instead it gives a link to google search.

I suggest either :
- to disable this feature
- or to create an infobox with the usable informations (names and phone numbers)

Related : this function doesn't to be call anymore : 
https://github.com/asciimoo/searx/blob/master/searx/engines/google.py#L220
It is triggered when you look for a specific location : a restaurant name for example.
I guess that the xpath are wrong : map_address_xpath, map_phone_xpath, map_website_url_xpath  and map_website_title_xpath

So to sum up, a quick fix is to remove the lines between 214 and 220.

**Comment by dalf** [comment]
Thank's !
yoooooyoooooyooooo :-)
